contentcuration/contentcuration/frontend/shared/views/channel/ChannelModal.vue

```diff
@@ -280,7 +280,10 @@
       ...mapActions('channel', ['updateChannel', 'loadChannel', 'commitChannel']),
       ...mapMutations('channel', ['REMOVE_CHANNEL']),
       saveChannel() {
-        if (this.$refs.detailsform.validate()) {
+        // the additional this.language check below is to account
+        // for a validation error caused in some browsers
+        // when a user set a default language in the old version of Studio
+        if (this.$refs.detailsform.validate() && this.language) {
```

Comment:
@MisRob - to be honest, I'm not totally sure! I'm doing a bit of guessing about where thing are going wrong, because I haven't yet figured out how to reproduce the error, since Radina's best guess is that the language was set default in her account in the old studio, and is still there, even though this feature is no longer part of the UI. I am trying to figure out how to configure an account that replicates hers for testing. 

In the meantime, here is what I'm thinking. My guess is that the language getter is returning a value 
`get() {
          return Object.prototype.hasOwnProperty.call(this.diffTracker, 'language')
            ? this.diffTracker.language
            : this.channel.language || this.currentLanguage;
        },` 

which is filling in the form, initially, but then the validation runs and for one reason or another, it isn't actually an accepted/valid value. I do recall [another issue where autofilled values caused similar validation errors](https://github.com/learningequality/studio/issues/2565). In both cases, you might a "flash" where the field is empty and the "required" UI appears, but this doesn't prevent the modal from closing, and then the error icon appears in the main view. 

So, my thoughts are that we should either try to prevent the language from autofilling, if that is causing the problem, which maybe would mean adding some conditional logic to the language property, or adding something else to "catch" the validation before the window closes. This was what I was attempting here, although now I am not sure that is really the ideal solution, because the user will always think the language field is complete, close, and have to go back and do it again. If this is an old setting that they cannot update, I imagine that might be very frustrating. I'm going to think on this 😄 and hopefully have some new ideas tomorrow!

Reply:
> creating "a user with a default language that had been previously set" so that I can test it. I am not familiar with the old Studio interface and can't find such an option in the new one.

@MisRob Since the _new_ Studio does not require (or offers to) users the option to specify the default channel language (but treats `Language` as a required field each time they create a channel), I'm not sure the above can be achieved through the current UI. That option did exist in the old Studio, among the user account Preferences settings:

![default-settings](https://user-images.githubusercontent.com/1457929/109083016-5cc6b400-7705-11eb-897d-a7f35f0ec8a9.png)

Reply:
Thank you, both. I've noticed @marcellamaki [message on Slack](https://learningequality.slack.com/archives/C0LK8QS9J/p1614211637049400) so I posted some suggestions to the thread.

Reply:
@marcellamaki Thank you for the explanation, this makes sense to me. Hopefully, we'll figure out why Radina's prefilled language is not accepted in the new Studio, if that's the case (from what I've seen in the code and posted on Slack, it might indeed be so).